scripts/windows/openclaude-aliases.ps1

```
@@ -0,0 +1,171 @@
+
+function oc {
+  [CmdletBinding()]
+  param(
+    [Parameter(ValueFromRemainingArguments = $true)]
+    [string[]]$OpenClaudeArgs
+  )
+
+  Invoke-OpenClaude @OpenClaudeArgs
+}
+
+function oc-local {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'oc-local' uses an unapproved verb.

(PSUseApprovedVerbs)